docs/content/faq/how-can-i-use-multiple-clusters-with-cli.md

```diff
@@ -0,0 +1,6 @@
+# How can I use multiple Flynn clusters with the CLI tool
+
+Flynn's CLI multiple clusters out of the box. Use the `-c` option to specify a cluster by name.
+
+    # List apps on a Flynn cluster named production
+    $ flynn -c production apps
```

Comment:
This should mention the behavior when using multiple remotes with different clusters (it just works, and you can specify the `-a` flag to the remote name to pick the app+cluster.